src/Symfony/Bridge/Doctrine/Listener/DoctrineConnectionListener.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Listener;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+final class DoctrineConnectionListener implements EventSubscriberInterface
+{
+    /**
+     * @param \ArrayObject<int, array{name: string, timing: int}> $connectionTiming
```

Comment:
```suggestion
     * @param \ArrayObject<name, int> $connectionEols
```

Reply:
you mean `\ArrayObject<string, int> $connectionExpiries` :)